flask_resty/testing.py

```diff
@@ -110,7 +110,7 @@ def assert_response(response, expected_status_code, expected_data=UNDEFINED):
     status_code = response.status_code
     assert status_code == expected_status_code
 
-    if response.content_length == 0:
+    if not response.content_length:
```

Comment:
this isn't 100% correct in all cases – content length of `None` could just mean it was omitted, but in practice this should be okay for the kinds of JSON requests that come out of flask resty